ansible-contrib/roles/master/files/delay-master-services.target

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Delay Kubernetes services until network is actually online
```

Comment:
What is the purpose of this unit?  Why not just put the dependencies into flanneld.service & kube-apiserver.service?